lib/rng.cppo.ml

```diff
@@ -0,0 +1,27 @@
+type t = ?base64:bool -> int -> string
```

Comment:
The only reason I didn't do this earlier is because nocrypto can base64 on cstructs directly, whereas doing it at the call site would lead to an extra allocation -- but only for nocrypto. Probably rather insignificant though...